docs/testplan/loss_threshold_test.md

```diff
@@ -0,0 +1,45 @@
+# SONiC Switch Loss Threshold Test
+
+- [SONiC Switch Loss Threshold Test](#sonic-switch-loss-threshold-test)
+  - [Test Objective](#test-objective)
+  - [Test Setup](#test-setup)
+  - [Test Steps](#test-steps)
+  - [Metrics Processing](#metrics-processing)
+
+## Test Objective
+
+This test aims to determine the maximum traffic rate that results in 0% packet loss across different packet sizes. By identifying this threshold, we can assess the switch's forwarding capability and validate its performance under various traffic conditions.
+
+## Test Setup
+
+This test is designed to be flexible and applicable to various network setups for evaluating traffic loss thresholds. It can be used to assess different configurations, such as one-tier and multi-tier networks. If testing the setup described in `BGP_IPv6_test.md`, ensure the testbed is properly configured by following the procedures outlined in that plan.
```

Comment:
update this to link.

also, better rename the BGP_IPv6_test.md to something like multi-tier-no-neighbor-testbed-setup.md